style.css

```diff
@@ -0,0 +1,375 @@
+/* Reset  */
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  margin-top: 5px;
+}
+
+body {
+  min-height: 100vh;
+  margin-left: 40px;
+  margin-right: 40px;
+  scroll-behavior: smooth;
+}
+
+/* Header Section  */
+
+.header-grid {
+  display: grid;
+  grid-template:
+    "twitter-logo home-grid search" auto /
+    auto 1fr auto;
+  gap: 5px;
+}
```

Comment:
Since we're creating a grid with a single column here I would suggest a more concise way of doing it:
`.header-grid {
  display: grid;
  grid-template-columns: auto 1fr auto;
  
  gap: 5px;
}`
This way there is no need to assign grid-areas later.